ZenPacks/zenoss/Import4/rrd2tsdb.py

```diff
@@ -158,7 +160,7 @@ def _process_v(self, tag_path, content):
         else:
             print '{} {:d} {:.10e} device={} key={} zenoss_tenant_id={}'.format(
                 self.metric, self.last_timestamp, _value,
-                self.device, self.key.replace(' ', '-'), self.dmd_uuid)
+                self.device, self.key.translate(self.tbl), self.dmd_uuid)
```

Comment:
The valid tag of opentsdb tag is actually [a-zA-Z.-_] (The white characters)
For the rest of the illegal ones, we map it to '-'.
I used the whole printable string just to show the exact mapping.